src/v2/BaseLoginRouter.js

```diff
@@ -32,11 +32,21 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 
-const introspectStateToken = (settings) => {
+const introspect = (settings) => {
+  const clientId = settings.get('clientId');
   const domain = settings.get('baseUrl');
+  const scopes = settings.get('scopes');
   const stateHandle = settings.get('stateToken');
+  const redirectUri = settings.get('redirectUri');
   const version = settings.get('apiVersion');
-  return idx.start({ domain, stateHandle, version });
+  
+  if (stateHandle) {
+    return idx.start({ domain, stateHandle, version });
+  } else if (settings.get('useInteractionCodeFlow')) {
+    return idx.start({ clientId, domain, scopes, stateHandle, redirectUri, version });
```

Comment:
This should be issuer, not domain, if we have issuer.  (the stateHandle version can continue to use domain, or can also switch over to issuer)